README.md

```diff
@@ -41,6 +41,10 @@ Besides the applications that are written in C, it is recommend to use Python
 to write tools or interfaces related to running tests. That allows for simple
 integration with the RF test scripts (and re-usability in general).
 
+## Testing a Pull Request
```

Comment:
I would rephrase (or generalise) here, bc its not only about testing a PR (which you btw. cannot simply checkout within the submodule without changing its git config, too). Also what PR? From RIOT-OS or RobotFW-tests 😮

So its more about testing a different version of RIOT, i.e. not the current submodule state. So you can either checkout that RIOT version somewhere else and overwrite `RIOTBASE` or update the RIOT submodule to the desired commit, however for the latter you cannot simply switch to a PR.

Reply:
Testing and Changing RIOT Versions